### ISSUE TITLE
About the pretrained model RN50-w5

### ISSUE THREAD

**Issue by hhb072** [facebookresearch/swav#35]
Thank you for sharing this wonderful work.

I conducted several experiments using the released pretrained models. However, the pretrained resnet50w5 is failed to be loaded because the batchnorm layer of the projection_head is missing in the pretrained models.

So could I just ignore this batchnorm layer when using the pretrained resnet50w5?

Thanks very much!

**Comment by mathildecaron31** [comment]
Hi @hhb072 ,
Thanks for your interest in our work. Yes indeed we trained the w5 model without batch norm in the head so you can just comment the following line when loading the model:
https://github.com/facebookresearch/swav/blob/b4aa051475144700d9532a61d9a61654ea236c52/src/resnet50.py#L207
